<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_trem1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: red}</string>
    </property>
    <property name="text">
     <string>T1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_liga">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>30</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Ligar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_para">
    <property name="geometry">
     <rect>
      <x>1015</x>
      <y>30</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Parar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="text">
     <string>T2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3_1">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3_2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>270</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_7">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_5">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>150</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: green}</string>
    </property>
    <property name="text">
     <string>T3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: purple}</string>
    </property>
    <property name="text">
     <string>T4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: orange}</string>
    </property>
    <property name="text">
     <string>T5</string>
    </property>
   </widget>
   <widget class="QSlider" name="t1_slider">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>100</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 0);</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_min">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>62</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_random">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>62</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Random</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_max">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>62</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
   </widget>
   <widget class="QSlider" name="t2_slider">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>120</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: 'blue';</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="t3_slider">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>140</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: 'green';</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="t4_slider">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>160</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: 'purple';</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="t5_slider">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>180</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: 'orange';</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_r1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_r2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_r3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_r4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_r5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>170</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_r6">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_r7">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R7</string>
    </property>
   </widget>
   <widget class="QLabel" name="veloci_1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 1: 0 ms/un</string>
    </property>
   </widget>
   <widget class="QLabel" name="veloci_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 2: 0 ms/uns</string>
    </property>
   </widget>
   <widget class="QLabel" name="veloci_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>50</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 3: 0 ms/un</string>
    </property>
   </widget>
   <widget class="QLabel" name="veloci_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 4: 0 ms/un</string>
    </property>
   </widget>
   <widget class="QLabel" name="veloci_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>250</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 5: 0 ms/un</string>
    </property>
   </widget>
   <zorder>label_trilho2_4</zorder>
   <zorder>label_trilho1_5</zorder>
   <zorder>label_trilho1_7</zorder>
   <zorder>label_trilho2_5</zorder>
   <zorder>label_trilho1_6</zorder>
   <zorder>label_trilho1_1</zorder>
   <zorder>label_trilho1_3</zorder>
   <zorder>label_trilho1_2</zorder>
   <zorder>label_trilho1_4</zorder>
   <zorder>label_trem1</zorder>
   <zorder>pb_liga</zorder>
   <zorder>pb_para</zorder>
   <zorder>label_trilho2_2</zorder>
   <zorder>label_trilho2_1</zorder>
   <zorder>label_trilho2_3</zorder>
   <zorder>label_trem2</zorder>
   <zorder>label_trilho3_3</zorder>
   <zorder>label_trilho3_1</zorder>
   <zorder>label_trilho3_2</zorder>
   <zorder>label_trem3</zorder>
   <zorder>label_trem4</zorder>
   <zorder>label_trem5</zorder>
   <zorder>t1_slider</zorder>
   <zorder>pb_min</zorder>
   <zorder>pb_random</zorder>
   <zorder>pb_max</zorder>
   <zorder>t2_slider</zorder>
   <zorder>t3_slider</zorder>
   <zorder>t4_slider</zorder>
   <zorder>t5_slider</zorder>
   <zorder>label_r1</zorder>
   <zorder>label_r2</zorder>
   <zorder>label_r3</zorder>
   <zorder>label_r4</zorder>
   <zorder>label_r5</zorder>
   <zorder>label_r6</zorder>
   <zorder>label_r7</zorder>
   <zorder>veloci_1</zorder>
   <zorder>veloci_2</zorder>
   <zorder>veloci_3</zorder>
   <zorder>veloci_4</zorder>
   <zorder>veloci_5</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
